src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -1645,6 +1645,34 @@ public bool Equals(StringBuilder sb)
             }
         }
 
+        /// <summary>
+        /// Determines if the contents of this builder are equal to the contents of ReadOnlySpan<char>.
+        /// </summary>
+        /// <param name="value">The ReadOnlySpan<char>.</param>
```

Comment:
This is invalid XML.  It should be `ReadOnlySpan{T}` rather than `ReadOnlySpan<char>`.